auth/service.cc

```diff
@@ -8,6 +8,7 @@
 
 #include <exception>
 #include <seastar/core/coroutine.hh>
+#include "auth/authenticator.hh"
```

Comment:
why added?

Reply:
Remnant of the previous version (the include also appears in a few other places). I'll remove it in the next version